pkg/reader/usdc_reader.go

```diff
@@ -109,11 +105,34 @@ func NewUSDCMessageReader(
 	return usdcMessageReader{
 		lggr:            lggr,
 		contractReaders: contractReaders,
-		cctpDestDomain:  CCTPDestDomains,
+		cctpDestDomain:  AllAvailableDomains(),
 		boundContracts:  boundContracts,
 	}, nil
 }
 
+// FIXME It adds test selectors to the domains
+func AllAvailableDomains() map[uint64]uint32 {
```

Comment:
This is temporary support for test domains. Ideally, we want to fetch these from the chain